### PR TITLE
Expose flower and redis ports in breeze

### PR DESCRIPTION
**Pull request by turbaszek**
This definitely helps with debugging CeleryExecutor.

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: scripts/ci/docker-compose/base.yml [hunk 1]
```diff
@@ -72,3 +72,4 @@ services:
       - /dev/urandom:/dev/random   # Required to get non-blocking entropy source
     ports:
       - "${WEBSERVER_HOST_PORT}:8080"
+      - "5555:5555" # Flower default port
```
Comment (turbaszek) [inline scripts/ci/docker-compose/base.yml]: @potiuk not sure if we need to make it configurable or not

Comment (gigkokman) [reply scripts/ci/docker-compose/base.yml]: It's flexible for user when port's in use.

Comment (turbaszek) [reply scripts/ci/docker-compose/base.yml]: True, same is for webserver port but we are still hardcoding it. I think this is ok for the development environment

**Review by potiuk — APPROVED** [review]
:heart:

File: scripts/ci/docker-compose/base.yml (hunk 1 above)
Comment (potiuk) [reply scripts/ci/docker-compose/base.yml]: Yep. Those are default ports, but you can always override them via the variables if for whatever reason you cannot use the default ones. That was the main reason why they are set as variables.

**Pull request merged**